app/furniture/marketplace/cart/delivery_area_component.html.erb

```diff
@@ -12,7 +12,7 @@
         <%= link_to(t('marketplace.cart.delivery_areas.edit.link_to'), cart.location(:edit, child: :delivery_area)) %>
       </div>
     <%- else %>
-      <%- if cart.marketplace.has_single_delivery_area? %>
+      <%- if single_delivery_area? %>
         <div><%= single_delivery_area_label %></div>
         <small>If you are ordering from outside this area then 🤷</small>
```

Comment:
```suggestion
        <small>Orders with delivery addresses outside of this delivery area will be subject to cancellation.</small>
```

Reply:
Now that I see it, "Orders outside of this location will be subject to cancellation." is even terser. Ugh writing for applications is hard.